### PR TITLE
Back button fix

### PR DESCRIPTION
**Pull request by alex-cory**

### CODE REVIEW

File: src/scenes/Event/scenes/oracle.js [hunk 1]
```diff
@@ -469,7 +467,7 @@ export default class OraclePage extends Component {
     const { locale, messages: localeMessages } = this.props.intl;
     const intl = getIntlProvider(locale, localeMessages);
 
-    if (!this.props.appLocation) this.props.setAppLocation(AppLocation.bet);
+    if (!this.props.appLocation) this.props.store.ui.location = AppLocation.bet;
```
Comment (dwalintukan) [inline src/scenes/Event/scenes/oracle.js]: shouldnt these check the mobx store location?
`if (!this.props.store.ui.location) ...`

File: src/scenes/Event/scenes/oracle.js [hunk 2]
```diff
@@ -488,7 +486,7 @@ export default class OraclePage extends Component {
   };
 
   setBetConfig = () => {
-    if (!this.props.appLocation) this.props.setAppLocation(AppLocation.bet);
+    if (!this.props.appLocation) this.props.store.ui.location = AppLocation.bet;
```
Comment (dwalintukan) [inline src/scenes/Event/scenes/oracle.js]: same comment ^

File: src/scenes/Event/scenes/oracle.js [hunk 3]
```diff
@@ -506,7 +504,7 @@ export default class OraclePage extends Component {
     const { locale, messages: localeMessages } = this.props.intl;
     const intl = getIntlProvider(locale, localeMessages);
 
-    if (!this.props.appLocation) this.props.setAppLocation(AppLocation.resultSet);
+    if (!this.props.appLocation) this.props.store.ui.location = AppLocation.resultSet;
```
Comment (dwalintukan) [inline src/scenes/Event/scenes/oracle.js]: same comment ^

File: src/scenes/Event/scenes/oracle.js [hunk 4]
```diff
@@ -528,7 +526,7 @@ export default class OraclePage extends Component {
     const { locale, messages: localeMessages } = this.props.intl;
     const intl = getIntlProvider(locale, localeMessages);
 
-    if (!this.props.appLocation) this.props.setAppLocation(AppLocation.vote);
+    if (!this.props.appLocation) this.props.store.ui.location = AppLocation.vote;
```
Comment (dwalintukan) [inline src/scenes/Event/scenes/oracle.js]: same comment ^

File: src/scenes/Event/scenes/oracle.js [hunk 5]
```diff
@@ -550,7 +548,7 @@ export default class OraclePage extends Component {
     const { locale, messages: localeMessages } = this.props.intl;
     const intl = getIntlProvider(locale, localeMessages);
 
-    if (!this.props.appLocation) this.props.setAppLocation(AppLocation.finalize);
+    if (!this.props.appLocation) this.props.store.ui.location = AppLocation.finalize;
```
Comment (dwalintukan) [inline src/scenes/Event/scenes/oracle.js]: same comment ^

File: src/redux/App/reducer.js [hunk 6]
```diff
@@ -31,9 +31,6 @@ const initState = new Map({
 
 export default function appReducer(state = initState, action) {
   switch (action.type) {
-    case actions.SET_APP_LOCATION: {
```
Comment (dwalintukan) [inline src/redux/App/reducer.js]: also dont forget to remove the default `appLocation`

File: src/scenes/Event/scenes/oracle.js (hunk 1 above)
Comment (alex-cory) [reply src/scenes/Event/scenes/oracle.js]: yes. will change

File: src/redux/App/reducer.js (hunk 6 above)
Comment (alex-cory) [reply src/redux/App/reducer.js]: fixed

**Pull request merged**